code/src/app.css

```diff
@@ -0,0 +1,16 @@
+*{
+    background-color: aliceblue;
+}
+
+@media (min-width: 450px){
+    .backgroundContainer{
+        border: 1.5px solid midnightblue;
+        width: 400px;
+        height: 550px;
+        margin-left: auto;
+        margin-right: auto;
+        margin-top: 10px;
+        border-radius: 10px;

```

Comment:
Add `overflow: scroll;`
To make this div scrollable when you have many todos ☺️